awx/ui/client/features/jobs/jobsList.controller.js

```diff
@@ -159,7 +159,7 @@ function ListJobsController (
     };
 
     vm.getWebhookDetails = (job) => {
-        if (!job.webhook_guid) {
+        if (job.launch_type !== 'webhook') {
```

Comment:
Nice. So I assume that the heuristic `!job.webhook_guid` would still work. Switching over to launch_type just makes it cleaner.

Reply:
Yep